adafruit_74hc595.py

```diff
@@ -101,15 +102,15 @@ def value(self, val: bool) -> None:
             self._shift_register.gpio = gpio
 
     @property
-    def direction(self) -> "typing.Optional[digitalio.Direction]":
+    def direction(self) -> digitalio.Direction.OUTPUT:
         """``Direction`` can only be set to ``OUTPUT``."""
         return digitalio.Direction.OUTPUT
 
     @direction.setter
     def direction(  # pylint: disable=no-self-use
         self,
         val: digitalio.Direction,
-    ) -> None:
+    ) -> "typing.Union[None, typing.NoReturn]":
```

Comment:
This should just be `digitalio.Direction.OUTPUT`, since any other `digitalio.Direction` causes an exception (by design).

Reply:
Also fixed the return type on this